packages/block-editor/src/components/navigable-toolbar/index.js

```diff
@@ -238,12 +238,7 @@ export default function NavigableToolbar( {
 	}
 
 	return (
-		<NavigableMenu
-			orientation="horizontal"
```

Comment:
I think we don't need that because orientation is already being passed inside props, I might be missing something here @jeryj

Reply:
I think we did remove it. It looks like the updated version is `role="toolbar" ref={ toolbarRef } { ...props }`. Maybe we should pass a default of `orientation: horizontal` for backwards compatibility though? In case they don't pass an orientation, it would no longer be there for older versions.

Reply:
Oh, actually we should leave it there. `<NavigableMenu />` has a default orientation of `vertical`. If we remove this, then anyone using this component that's not passing an orientation will have their toolbar orientation switch from `horizontal` to `vertical`.